packages/react-button/src/components/SplitButton/SplitButton.test.tsx

```diff
@@ -5,6 +5,7 @@ import { isConformant } from '../../common/isConformant';
 describe('SplitButton', () => {
   isConformant({
     Component: SplitButton,
+    disabledTests: ['as-renders-fc', 'as-renders-html'],
```

Comment:
**q:** can pls you briefly summarise why are all those changes needed for isConformant calls ? thx

Reply:
`useSplitButton` implements `mergeProps()` in a different way:

https://github.com/microsoft/fluentui/blob/master/packages/react-button/src/components/SplitButton/useSplitButton.ts#L18-L82

`as` prop is not really passed and is always overridden with `as: 'span'` 😨

Reply:
huh not sure I follow, I guess I'll have to dig deeper in the whole `isConformant` api. thx anyways!